### DISCUSSION TITLE
Please let me know github

### DISCUSSION THREAD

**Discussion by solyasa** [community/community#31684] [Other Feature Feedback, Questions, & Ideas]
Hello everyone
I saw this issue. this is github issue or my issue?
Let me know about that.
Best regards

![github](https://user-images.githubusercontent.com/97841807/188044171-4885a418-9f11-456b-9b41-1e807c93c049.png)

**Comment by softorangetech1122** [comment]
Maybe Primer CSS made that issue... (Primer is the design system that powers GitHub)

**Comment by melvincwng** [comment]
@Gould219 Hi there, this is a known issue in GitHub.

A bug report already has been made, and the support engineers should already be looking to resolve this issue.

Reference: https://github.com/community/community/discussions/30938

Hope this answers your question 😊

**Reply by solyasa** [reply]
Can you show me detail report?

**Reply by radar-blip** [reply]
This issue/bug showed up about a week ago, its known and reported. 👍🏻 
And should be fixed soon.

**Reply by solyasa** [reply]
Okay
can you send me report url?

**Reply by radar-blip** [reply]
Im sorry, I dont have a report but if you read the discussion referenced to you by melvincwng, 
you will see the issue has been reported and also a reponse from Github support.

If it helps at all, refresh your page and the dates should return to full view. 
Dont worry, its not your computer.
